obal/data/roles/build_srpm/defaults/main.yaml

```diff
@@ -0,0 +1,5 @@
+---
+build_srpm_tito_releaser_args: []
+build_srpm_dist_args: ''
```

Comment:
what is this (and the next one) for? I don't see the vars used.

Reply:
removed